src/main/java/com/mineinabyss/mobzy/registration/MobzyTypeRegistry.kt

```diff
@@ -81,7 +81,8 @@ object MobzyTypeRegistry {
         "mobzy:hostile" to ::HostileMob,
         "mobzy:passive" to ::PassiveMob,
         "mobzy:fish" to ::FishMob,
-        "mobzy:npc" to ::NPC
+        "mobzy:npc" to ::NPC,
+        "mobzy:projectile" to ::ProjectileEntity //TODO should not be necessary after entity refactoring
```

Comment:
Will be removed after refactoring projectile to Entity instead of CustomMob

Reply:
It'll still be necessary, I havent touched this part of the system at all though I'd like to